packages/medusa/src/strategies/batch-jobs/price-list/import.ts

```diff
@@ -312,7 +312,7 @@ class PriceListImportStrategy extends AbstractBatchJobStrategy {
             (op.prices as PriceListPriceCreateInput[]).map((p) => {
               return {
                 ...p,
-                variant_id: op.variant_id,
+                variant_id: op.variant_id.toString(),
```

Comment:
not exactly since we are not referencing `variant_id` from `p` but instead `op` which is just a record with key-values of type `string | number | (string | number | object)[]`. For now I've cast it as string instead

@fPolic is it intentional that it's not referencing the specific row but instead the parent row?

Reply:
I am fine with keeping it as it is guys